datastore/google/cloud/datastore/batch.py

```diff
@@ -133,7 +135,8 @@ def _add_delete_key_pb(self):
         :returns: The newly created key protobuf that will be
                   deleted when sent with a commit.
         """
-        new_mutation = self.mutations.add()
+        new_mutation = _datastore_pb2.Mutation()
+        self._mutations.append(new_mutation)
         return new_mutation.delete
 
     @property
```

Comment:
Since this is just a 'getter' and it's *just* returning `self._mutations`, could this property be removed?

I'm not sure why someone would need to use it practically?